data/856/829/89/85682989.geojson

```diff
@@ -104,20 +110,18 @@
         "\u0395\u03c0\u03b1\u03c1\u03c7\u03af\u03b1 \u03a7\u03b9\u03af\u03bf\u03c5"
     ],
     "name:eng_x_preferred":[
-        "Hiiumaa"
-    ],
-    "name:eng_x_variant":[
-        "Hiiu County",
         "Hiiu"
     ],
-    "name:epo_x_preferred":[
-        "Hiiumaa"
+    "name:eng_x_variant":[
+        "Hiiu County"
```

Comment:
Why isn't `Hiiumaa` listed as English variant?

Reply:
This is a Estonian variant of the name (stored in `"name:est_x_variant"`).